### ISSUE TITLE
SG - Vault Secret Service - Access Model and Rollout

### ISSUE THREAD

**Issue by mitovskaol** [BCDevOps/developer-experience#967]
The purpose of this goal is to finish the Vault Access Model enabling select users on teams to login to vault and edit secrets. Once the model is complete it will be rolled into Silver for the platform services team to verify functionality and create additional documentation for teams. 

Stretch goal of enabling early access for teams outside of the platform services team at the end of the sprint.

**Comment by manderson-it** [comment]
@mitovskaol @j-pye Justin and I had a working session last week Thursday to simulate the onboarding experience of a new Vault consumer. Justin, let me know if you want me to write the docs from your notes or would like proof-reading etc.

**Comment by j-pye** [comment]
Due to discussions around our use of Vault last week, this work will continue into the next sprint. Changes may need to be made based on discussion in an upcoming meeting on April 8th.

**Comment by j-pye** [comment]
Complete.
